test_df/test_data_functions.py

```diff
@@ -199,12 +199,42 @@ def test_named_data_function_translate_sequences(self) -> None:
 
     def test_translate_sequences_script(self) -> None:
         from test_df.translate_sequences_script import execute
-        file_in = os.path.join(os.path.dirname(__file__), 'resources', 'translate_sequences_script.json')
-        _, response = run_script(file_in, execute)
+        # file_in = os.path.join(os.path.dirname(__file__), 'resources', 'translate_sequences_script.json')

```

Comment:
I'll remove these commented lines once the PR is approved.  I didn't want to completely lose the previous test code.

Reply:
Removed commented code - can always roll back if needed.